docs/handbook/how-to-become-a-player.md

```diff
@@ -8,7 +8,7 @@ So you've been tumbling down this rabbit hole & are wondering how to actually st
 Well alright! Let's get you geared up! :cowboy_hat_face:
```

Comment:
This cowboy hat face isn't actually rendering as an emoji
![image](https://user-images.githubusercontent.com/7143583/85641406-b729e500-b64b-11ea-94e9-33f60849f3da.png)